tests/Command/Issue/IssueTakeCommandTest.php

```diff
@@ -54,33 +55,16 @@ public function takes_an_issue_with_specific_base()
 
         $tester = $this->getCommandTester($command = new IssueTakeCommand());
         $command->getHelperSet()->set($this->expectTextHelper());
-        $command->getHelperSet()->set($this->expectGitHelper('origin', 'origin/development'));
+        $command->getHelperSet()->set($this->expectGitConfigHelper());
+        $command->getHelperSet()->set($this->expectGitHelper('gushphp', 'gushphp/development'));
 
         $tester->execute(
-            ['--org' => 'gushphp', 'issue_number' => TestAdapter::ISSUE_NUMBER, 'base_branch' => 'development'],
-            ['interactive' => false]
-        );
-
-        $this->assertEquals(
-            sprintf('[OK] Issue https://github.com/gushphp/gush/issues/%s taken!', TestAdapter::ISSUE_NUMBER),
-            trim($tester->getDisplay(true))
-        );
-    }
-
-    /**
-     * @test
-     */
-    public function takes_an_issue_with_specific_remote()
-    {
-        $this->expectsConfig();
-        $this->config->get('base')->willReturn('master');
-
-        $tester = $this->getCommandTester($command = new IssueTakeCommand());
-        $command->getHelperSet()->set($this->expectTextHelper());
-        $command->getHelperSet()->set($this->expectGitHelper('gushphp', 'gushphp/master'));
-
-        $tester->execute(
-            ['--org' => 'gushphp', 'issue_number' => TestAdapter::ISSUE_NUMBER, '--remote' => 'gushphp'],
+            [
+                '--org' => 'gushphp',
+                '--repo' => 'gush',
+                'issue_number' => TestAdapter::ISSUE_NUMBER,
+                'base_branch' => 'development'
```

Comment:
what i am seeing though say for example in "el puli" cc @webmozart is that he has centralized all the issues in a repository out of the many repos that currently exist. This breaks the schema of things though because now we take issues from a repository but we send PRs to other repositories.